### ISSUE TITLE
[UX2.0] Parcel state refresh fails when parcel was deleted in UI

### ISSUE THREAD

**Issue by tzarski0** [CiscoDevNet/terraform-provider-sdwan#291]
1. Create resources and do terraform apply:
```
resource "sdwan_cli_feature_profile" "cli_feature_profile" {
  name        = "cli"
  description = "test"
}

resource "sdwan_cli_config_profile_parcel" "cli_config_profile_parcel" {
  name               = "cli_config"
  description        = "test"
  cli_configuration  = "no logging console"
  feature_profile_id = sdwan_cli_feature_profile.cli_feature_profile.id
}
```

2. Go to UI https://<manager-ip>/#/app/config/configurationGroups/featureProfiles and delete profile "cli".

3. Run terraform apply again. It fails with:
```
│ Error: Client Error
│ 
│   with sdwan_cli_config_profile_parcel.cli_config_profile_parcel,
│   on main.tf line 21, in resource "sdwan_cli_config_profile_parcel" "cli_config_profile_parcel":
│   21: resource "sdwan_cli_config_profile_parcel" "cli_config_profile_parcel" {
│ 
│ Failed to retrieve object (GET), got error: HTTP Request failed: StatusCode 400,
│ {"error":{"message":"Invalid feature Id","code":"PPARC0009","details":"Invalid profile parcel id
│ 1af3a4d5-64b5-42d0-a3f4-21502010349b","type":"error"}}
```

**Comment by danischm** [comment]
Fix: https://github.com/CiscoDevNet/terraform-provider-sdwan/commit/b35a25ec9d8161d83fee0c5938b27dfe446838db

**Comment by tzarski0** [comment]
Verified and will be fixed in 0.3.14.
